### DISCUSSION TITLE
spaced repetition

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#34] [主发布站的评论系统]
# spaced repetition

间隔重复记忆。理论基础是[[艾宾浩斯记忆曲线]]。
市场上的[[🐤OG]]是[[anki]]和[[supermemo]]。
之前感觉一直没人开发竞品，可能这东西核心的是算法，软件界面倒是其次。而且商业化也不明朗，只有个人开发者。现在市场多了一些大玩家。墨墨背单词开发的markji，太记等等。
另一种存在形态，被[[🧑Andy Matuschak]]称为[[助记媒介]]。不单纯是记忆闪卡本身，还能融合到其他例如笔记软件中，进行有机地结合。
就像使用[[spaced repetition]]插件和obsidian完美结合一样。
 参考文献  Why I Switched to Obsidian After Using SuperMemo for 4 Years - Master How To Learn

https://garden.oldwinter.top/spaced-repetition/

**Comment by L-M-Sherlock** [comment]
其实核心也倒不是算法，而是怎么把间隔重复更好地融入到原有的学习流程中。渐进阅读和助记媒介都尝试将间隔重复融入到阅读中。
